### PR TITLE
Update k9

### PR DESCRIPTION
**Pull request by iuljan**
This PR contains the customized configs for the hardware change at location k9.

A 60GHZ antenna is aligned to Sama
And the previous one is aligned to Zwingli

Loge and Größenwahn get core APs (Aruba AP11) via LAN.

### CODE REVIEW

File: locations/k9.yml [hunk 1]
```diff
@@ -113,19 +74,37 @@ networks:
...
   - vid: 42
     role: mgmt
+    name: omgmt
     prefix: 10.31.9.208/28
     gateway: 1
     dns: 1
     ipv6_subprefix: 1
     assignments:
       k9-core: 1      # 10.31.9.209
       # k9-switch: 2  # 10.31.9.210
```
Comment (nailend) [inline locations/k9.yml:84]: @spolack can the old MGMT be removed?

Comment (nailend) [reply locations/k9.yml:84]: @Noki

Comment (Noki) [reply locations/k9.yml:84]: Yes, it can be removed. Everything is configured to use the new one.

**Comment by nailend** [comment]
replaced by #1121

**Pull request closed without merging**